docs/en/latest/FAQ.md

```diff
@@ -0,0 +1,52 @@
+---
+title: FAQ
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+### 1. How to install APISIX only?
+
+The Charts will install etcd 3.4.14 by default. If you want to install Apache APISIX only, please set `etcd.enabled=false` and set `etcd.host={http://your_etcd_address:2379}`.
+
+Please use the FQDN address or the IP of the etcd.
```

Comment:
```suggestion
Please use the FQDN or the IP address of the ETCD cluster.
```

Reply:
Please give your reason to fold this conversation.

Reply:
@tokers  Oh No..once I commit the suggested changes it gets marked as resolved the conversation automatically..